### ISSUE TITLE
iFrame Title for Accessibility

### ISSUE THREAD

**Issue by TwisterMc** [BenjaminLawson/PDF.js-Viewer-Shortcode#1]
For accessibility, iFrames should have titles. https://www.w3.org/TR/WCAG20-TECHS/H64.html

The user should be able to pass in a title with a generic title as a fallback.

**Comment by richaber** [comment]
@TwisterMc what would be a preferable "generic title as a fallback?"

**Comment by richaber** [comment]
We could use something like "Embedded PDF document" or "Embedded PDF viewer," if content author does not supply a title attribute, I'm just not sure what the preference would be.

**Comment by richaber** [comment]
@BenjaminLawson, @TwisterMc I'm going with 'Embedded PDF Document' as generic fallback for now, we can change it later if there is a desire to do so.

**Comment by TwisterMc** [comment]
I like "Embedded PDF Document" fas the fallback.
